### PR TITLE
Use gorilla/websocket for proxying websocket

### PR DESCRIPTION
**Pull request by juliens**

### CODE REVIEW

**Review by ldez — APPROVED** [review]
LGTM

**Review by dtomcej — APPROVED** [review]
This looks pretty sweet. Hopefully it resolves our WS issues moving forward

File: forward/fwd.go [hunk 1]
```diff
@@ -255,77 +253,40 @@ func (f *httpForwarder) copyRequest(req *http.Request, u *url.URL) *http.Request
...
-
-	// if host does not specify a port, use the default http port
-	if !strings.Contains(host, ":") {
-		if outReq.URL.Scheme == "wss" {
-			host = host + ":443"
-		} else {
-			host = host + ":80"
-		}
-	}
+	dialer := websocket.DefaultDialer
```
Comment (emilevauge) [inline forward/fwd.go]: Shouldn't we reuse the TLS Config in case of `wss` in the dialer ?

**Review by emilevauge — APPROVED** [review]
Thanks @Juliens 
LGTM !

**Pull request merged**